zerver/actions/invites.py

```diff
@@ -399,11 +399,26 @@ def do_get_invites_controlled_by_user(user_profile: UserProfile) -> List[Dict[st
                 ),
                 invited_as=invite.invited_as,
                 is_multiuse=True,
+                stream_ids=list(invite.streams.values_list("id", flat=True)),
```

Comment:
It looks kind of odd to me to only return `stream_ids` for multi-use invite links. I think it would make sense to return them for email invitations as well and the API changelog should be updated accordingly.

Reply:
Yeah, agreed.